hugegraph-llm/src/hugegraph_llm/utils/embedding_utils.py

```
@@ -0,0 +1,53 @@
+
+
+import asyncio
+from typing import Any
+
+from tqdm import tqdm
+
+from hugegraph_llm.models.embeddings.base import BaseEmbedding
+
+
+async def get_embeddings_parallel(embedding: BaseEmbedding, vids: list[str]) -> list[Any]:
+        sem = asyncio.Semaphore(10)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 8 spaces, expected 4

(W0311)